appserver/monitoring-console/webapp/pom.xml

```diff
@@ -94,4 +94,35 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+    
+    <build>
+        <plugins>
+            <plugin>
+              <groupId>com.bekioui.maven.plugin</groupId>
+              <artifactId>merge-maven-plugin</artifactId>
```

Comment:
We try to avoid bringing in third-party dependencies and plugins when they aren't strictly necessary.
Is there specifically a reason that you need this plugin to "merge" these files?

License seems to be fine as its Apache, just want to hear your reasoning :)

Reply:
There is no particular reason other than finding a platform independent way of merging the JS files into one. Having everything in one file got out of hand length wise so I split it to multiple files that are merged. But I also didn't want to pull in the full JS build stack madness so I found the merge plugin that does everything I need for now. Maybe there is a better way that I just have not thought about?

Reply:
@Pandrex247 Are you happy with the use of the plugin or should we look into how to else solve the issue? I'd imagine a web-server might have some feature to serve a virtual file composed of a list of resource files? :)